### PR TITLE
Add publish-queue, add inspect-queue --secrets-dir

### PR DESCRIPTION
**Pull request by martinpitt**
This CLI tool will allow us to write workflows that interact with our
tests, for example submit `store-tests` command to the statistics queue
for daily master tests.

-----
This is a prerequisite for both [daily master test runs](https://issues.redhat.com/browse/COCKPIT-751), as well as queueing integration tests after the "build-dist" workflow.

The two working together:
```
$ echo 'echo hello' | ./publish-queue -q statistics --secrets-dir ~/redhat/ci-secrets/webhook/

$ ./inspect-queue --secrets-dir ~/redhat/ci-secrets/webhook/ 
public queue:
queue public is empty
rhel queue:
queue rhel is empty
statistics queue:
{"command": "./store-tests --db /cache/images/test-results.db --repo cockpit-project/cockpit 8592582f455a4c67b0a431440b64649c43df12a3"}
{"command": "echo hello"}
```

The store-tests is of course from some test; the publish-queue entry is from the command above.

 - [x] Fix run-local.sh to not rely on internal default `--amqp`: https://github.com/cockpit-project/cockpituous/pull/386

### CODE REVIEW

**Comment by martinpitt** [comment]
Submitting this to origin, as this touches inspect-queue, and we should do the full "cockpituous" test glory.

**Comment by martinpitt** [comment]
@Gundersanne FYI -- I guess you also use inspect-queue from time to time, and that makes it a lot more convenient.

**Comment by martinpitt** [comment]
Hmm, we were a bit shy about hardcoding amqp-frontdoor.apps.ocp.ci.centos.org as a default; but "localhost" makes sense almost nowhere, and like that we would have to hardcode it in several cockpit PRs. So let me rework this to have a sensible --amqp default.

**Comment by martinpitt** [comment]
There, that's better. With that we can write a sensible workflow for submitting PR statistics, instead of [this hack](https://github.com/martinpitt/cockpit/blob/tests-trigger-workflow/.github/workflows/pr-statistics.yml#L29).

**Comment by martinpitt** [comment]
Meh, this breaks -- need to call `inspect-queue` with the right local `--amqp`. I sent https://github.com/cockpit-project/cockpituous/pull/386 to fix this.

File: task/distributed_queue.py [hunk 1]
```diff
@@ -72,13 +79,16 @@ def __init__(self, amqp_server, queues, **kwargs):
         if no_amqp:
             raise ImportError('pika is not available')
 
+        if secrets_dir is None:
+            secrets_dir = DEFAULT_SECRETS_DIR
```
Comment (marusak) [inline task/distributed_queue.py:85]: Why not just use `DEFAULT_SECRETS_DIR` as default value in `__init__` definition?

**Review by marusak — APPROVED** [review]
Super nice, thanks!
One tiny nitpick. Approving in the case I missed reasoning behind it.

File: task/distributed_queue.py (hunk 1 above)
Comment (martinpitt) [reply task/distributed_queue.py:85]: You are right, this is nicer. Fixed.

**Review by marusak — APPROVED** [review]
Thanks!

**Pull request merged**